client/valor/exceptions.py

```diff
@@ -18,3 +23,77 @@ def __init__(self):
 class ClientConnectionFailed(Exception):
     def __init__(self, msg: str):
         super().__init__(msg)
+
+
+class ServiceUnavailable(ClientException):
```

Comment:
would it make sense to copy the docstrings from the `/api` exception classes to these classes for clarity? The docstring for this one would be "Raises an exception if the Valor service is unavailable."

Reply:
ya, will add